packages/ptokens-assets-evm/test/ptokens-utils-evm.test.ts

```diff
@@ -98,18 +62,21 @@ describe('ethereum utilities', () => {
   })
 
   test('Should get operation ID from log', async () => {
-    const res = await Promise.allSettled(
-      logs.map((_log) => new Promise((_resolve) => _resolve(utils.getOperationIdFromLog(_log)))),
+    const res_1 = await Promise.allSettled(
+      logs.slice(0, 3).map(
+        (_log) => new Promise((_resolve) => _resolve(utils.getOperationIdFromLog(_log as unknown as Log<bigint>, '0xf9b459a1' as NetworkId))),
+      ),
     )
+    const res_2 = await Promise.allSettled(
```

Comment:
Why 2 res? Just create a new test `Should get operation ID from log 2`

Reply:
Ah no, logs.map should be enough. Then at the end you check the promise allSettlet as it was earlier.